src/components/Header/Heading.js

```diff
@@ -63,7 +64,7 @@ Heading.propTypes = {
     medium: PropTypes.oneOf(Object.keys(typography.size)),
     large: PropTypes.oneOf(Object.keys(typography.size))
   }),
-  weight: PropTypes.oneOf(["light", "regular", "extraBold"]),
+  weight: PropTypes.oneOf(Object.keys(typography.weight)),
```

Comment:
@faichuk I think it will be nice to add these as the typescript types to our constatns, and potentially not enumerate the EXTRA_BOLD font-weight, so whoever is using it wrong gets a warning. e.g. button variants: https://github.com/ticketmaster/aurora/blob/master/src/components/constants.tsx#L27-L45

```javascript
export const FONT_WEIGHT_LIGHT = "light";
export const FONT_WEIGHT_REGULAR = "regular";
export const FONT_WEIGHT_SEMI_BOLD = "semiBold";

/** @deprecated: Use `FONT_WEIGHT_SEMI_BOLD` instead */
export const EXTRA_BOLD = "extraBold"; // <-- This is here for backward compatibility

export const FONT_WEIGHTS: ReadonlyArray<FontWeight> = [
  FONT_WEIGHT_LIGHT,
  FONT_WEIGHT_REGULAR,
  FONT_WEIGHT_SEMI_BOLD
];
export type FontWeight =
  | typeof FONT_WEIGHT_LIGHT
  | typeof FONT_WEIGHT_REGULAR
  | typeof FONT_WEIGHT_SEMI_BOLD;
```

Reply:
Done.